custom_components/entsoe/sensor.py

```diff
@@ -38,10 +57,13 @@ class EntsoeSensor(CoordinatorEntity, SensorEntity):
     _attr_attribution = ATTRIBUTION
     _attr_icon = ICON
 
-    def __init__(self, coordinator: EntsoeCoordinator, description: EntsoeEntityDescription) -> None:
+    def __init__(self, coordinator: EntsoeCoordinator, description: EntsoeEntityDescription, prefix: str = "") -> None:
         """Initialize the sensor."""
+
+        self.entity_id = generate_entity_id( ENTITY_ID_FORMAT, f"{prefix+description.name}", hass=coordinator.hass )
```

Comment:
In order to not have a breaking change this and the `attr_unique_id` needs to fallback to not being defined and the old `f"entsoe.{description.key}"` if prefix is empty.